pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -40,10 +40,6 @@
         <Exclude>Compile</Exclude>
       </Dependency>
     </ItemGroup>
-
-    <PropertyGroup>
-    <VersionSuffix>-rc3-$(CLIBuildVersion)</VersionSuffix>
```

Comment:
:( so you mean NETCore.App will now get that `-00` in its version number too?

Let's avoid that because we are using commits.  There will never be a need to change `-00`.